common/README.md

```diff
@@ -43,6 +43,8 @@ This postamble can be modified by setting the `prompt_postamble` variable in `sh
 **Supporting additional models**
 
 We natively provide support for the following models:
+- GPT-4o-mini (`gpt-4o-mini`)
```

Comment:
may want to point to the static name `gpt-4o-mini-2024-07-18`

Reply:
Updated the PR to point to the static version `gpt-4o-mini-2024-07-18`.

Thanks for the suggestion!